docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md

```diff
@@ -613,13 +613,13 @@ To install the YugabyteDB node agent manually, as the `yugabyte` user, do the fo
 1. Run the following command to download the node agent's `.tgz` file which installs and starts the interactive configuration:
 
    ```sh
-   ./installer.sh -c install -u https://<yba_address>:9000 -t <api_token>
+   ./installer.sh -c install -u https://<yba_address> -t <api_token>
    ```
 
    For example, if you run the following:
 
    ```sh
-   ./installer.sh  -c install -u http://10.98.0.42:9000 -t 301fc382-cf06-4a1b-b5ef-0c8c45273aef
+   ./installer.sh  -c install -u http://10.98.0.42 -t 301fc382-cf06-4a1b-b5ef-0c8c45273aef
```

Comment:
Let's change this to https for this as well. I think it is better to add a line below. 

To ignore verification of YBA certificate, --skip_verify_cert can be specified.

Reply:
@nkhogen - added it as a comment in the code block. Please verify.